src/Metadata/Property/PropertyMetadata.php

```diff
@@ -43,6 +44,7 @@ public function __construct(Type $type = null, string $description = null, bool
         $this->identifier = $identifier;
         $this->iri = $iri;
         $this->attributes = $attributes;
+        $this->childInherited = null;
```

Comment:
It should be kept internally handled IMO. Also, I wanted to avoid a bc break (logic would want to set it before `attributes` no?)